docker/Makefile

```diff
@@ -8,6 +8,7 @@ hadolint_container := hadolint/hadolint:latest
 export BUNDLE_PATH = $(PWD)/.bundle/gems
 export BUNDLE_BIN = $(PWD)/.bundle/bin
 export GEMFILE = $(PWD)/Gemfile
+export DOCKER_BUILDKIT = 1
```

Comment:
Was this ENV supposed to be referenced by your other changes on lines 34 and 36?

Reply:
Nope, it's a global setting